### PR TITLE
Fix some issues found by Denian's lintian tool

### LINKED ISSUE #6473: manpage warnings
Debian's lintian tool generates the following warning:

- usr/share/man/man3/SSL_CTX_set_cipher_list.3ssl.gz 179: warning [p 1, 6.8i]: can't break line
- usr/share/man/man1/ciphers.1ssl.gz 230: warning [p 2, 2.2i]: can't break line
- usr/share/man/man1/ciphers.1ssl.gz 230: warning [p 2, 2.2i]: can't break line
- usr/share/man/man1/dsa.1ssl.gz 212: warning [p 1, 10.0i, div `an-div', 0.0i]: can't break line
- usr/share/man/man1/dsa.1ssl.gz 212: warning [p 1, 10.0i, div `an-div', 0.0i]: can't break line
- usr/share/man/man1/gendsa.1ssl.gz 173: warning [p 1, 4.5i, div `an-div', 0.0i]: can't break line
- usr/share/man/man1/gendsa.1ssl.gz 173: warning [p 1, 4.5i, div `an-div', 0.0i]: can't break line
- usr/share/man/man1/genrsa.1ssl.gz 180: warning [p 1, 4.7i, div `an-div', 0.0i]: can't break line
- usr/share/man/man1/genrsa.1ssl.gz 180: warning [p 1, 4.7i, div `an-div', 0.0i]: can't break line
- usr/share/man/man1/rsa.1ssl.gz 213: warning [p 1, 9.7i, div `an-div', 0.0i]: can't break line
- usr/share/man/man1/rsa.1ssl.gz 213: warning [p 1, 9.7i, div `an-div', 0.0i]: can't break line

I understand it's generated by:
`LC_ALL=en_US.UTF-8 MANROFFSEQ='' MANWIDTH=80 man --warnings -E UTF-8 -l -Tutf8 -Z <file> >/dev/null`

### PR DESCRIPTION
**Pull request by richsalz**
Fixes #6473.  Started using the technique @levitte suggested, and then just did all of them.

A separate PR might happen for the 'man3' pages, which are more invasive.  This just fixes man[157]

I am not going to backport this.

### CODE REVIEW

File: doc/man1/ciphers.pod [hunk 1]
```diff
@@ -113,7 +113,7 @@ TLSv1.2 and below ciphersuites that have been configured. The format for this
 list is a simple colon (":") separated list of TLSv1.3 ciphersuite names. By
 default this value is:
 
-"TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256"
+  TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256
```
Comment (levitte) [inline doc/man1/ciphers.pod]: One space is enough...

File: doc/man1/ts.pod [hunk 2]
```diff
@@ -557,10 +557,10 @@ OID section of the config file):
 
 Before generating a response a signing certificate must be created for
 the TSA that contains the B<timeStamping> critical extended key usage extension
-without any other key usage extensions. You can add the
-'extendedKeyUsage = critical,timeStamping' line to the user certificate section
-of the config file to generate a proper certificate. See L<req(1)>,
-L<ca(1)>, L<x509(1)> for instructions. The examples
+without any other key usage extensions. You can add this line to the
+user certificate section of the config file to generate a proper certificate;
+   extendedKeyUsage = critical,timeStamping
```
Comment (levitte) [inline doc/man1/ts.pod]: You must have a blank line before and after this one. Remember, blank lines are block separators in the pod format.

**Comment by mspncp** [comment]
Nit:  s/Denian/Debian/g   (commit message and pr title)

File: doc/man5/config.pod [hunk 3]
```diff
@@ -64,8 +64,8 @@ length after variable expansion. Otherwise an error will occur.
 
 It is possible to escape certain characters by using any kind of quote
 or the B<\> character. By making the last character of a line a B<\>
-a B<value> string can be spread across multiple lines. In addition
-the sequences B<\n>, B<\r>, B<\b> and B<\t> are recognized.
+a B<value> string can be spread across multiple lines. In addition, a
+B<\> can be followed by one of B<n>, B<r>, B<b>, or B<t>.
```
Comment (levitte) [inline doc/man5/config.pod]: I'm not sure that this particular change was for the better...

File: doc/man5/config.pod [hunk 4]
```diff
@@ -308,7 +308,7 @@ mentioned above.
  cover several lines \
  by including \\ characters
 
- message = Hello World\n
+ message = Hello World\\n
```
Comment (levitte) [inline doc/man5/config.pod]: No, this isn't correct.

File: doc/man5/x509v3_config.pod [hunk 5]
```diff
@@ -180,7 +180,8 @@ L<ASN1_generate_nconf(3)> format.
 
 Examples:
 
- subjectAltName=email:copy,email:my@other.address,URI:http://my.url.here/
+ subjectAltName=email:my@other.address
+ subjectAltName=email:copy,URI:http://my.url.here/
```
Comment (levitte) [inline doc/man5/x509v3_config.pod]: Is there a reason for this?

Comment (richsalz) [reply doc/man5/x509v3_config.pod]: It's not incorrect.  It fixes a linewrap complaint.  Having multiple email addresses in the SAN is confusing.  If you want I will revert that.

File: doc/man5/config.pod (hunk 4 above)
Comment (richsalz) [reply doc/man5/config.pod]: look at line 309 above.  are you sure this is wrong?

File: doc/man5/config.pod (hunk 3 above)
Comment (richsalz) [reply doc/man5/config.pod]: Well, backslash-char means special things to pod and man.  And it also describes the way the code works better.  Still not okay?

Comment (levitte) [reply doc/man5/config.pod]: I don't see how that describes things better. Regarding backslashes, they have no general special meaning in the pod format (see `man perlpod`). `pod2man` converts them to `\e`, so roff converters will display them correctly.

Did you try `perl doc doc/man5/config.pod`? If you want to see the roff produced, you can give it `-n less`. That's a good way to check your claims...

File: doc/man5/config.pod (hunk 4 above)
Comment (levitte) [reply doc/man5/config.pod]: The point is what gets embedded into the conf values. You've just changed an embedded line feed to a backslash followed by `n`...

File: doc/man5/x509v3_config.pod (hunk 5 above)
Comment (levitte) [reply doc/man5/x509v3_config.pod]: I don't see this mentioned in #6473, so I assume you found this on your own. My `perldoc doc/man5/x509v3_config.pod` run does *not* linewrap this line, hence my surprise...

Comment (richsalz) [reply doc/man5/x509v3_config.pod]: Yes, I ran the lintian script equivalent given there by myself.

**Comment by levitte** [comment]
@richsalz, how exactly did you produce the manpages? I just did that with the section 5 pods, then tried the `man` command from #6473 on the results, and got no errors at all... I would like to see your process to try and reproduce it.

**Comment by richsalz** [comment]
I used the command line mentioned there, wrapped in a script.  My man version is 2.6.7.1

**Comment by levitte** [comment]
Ok, so let me show you mine:

```
: levitte@lapdog:~/gitwrk/openssl.net/official/_build
: ; (cd ../master/; LANG=C git pull --rebase)
Already up to date.
Current branch master is up to date.
: ; man --version
man 2.8.3
: ; ../master/config 
Operating system: x86_64-whatever-linux2
Configuring OpenSSL version 1.1.1-pre9-dev (0x10101009L) for linux-x86_64
Using os-specific seed configuration
Creating configdata.pm
Creating Makefile

**********************************************************************
***                                                                ***
***   If you want to report a building issue, please include the   ***
***   output from this command:                                    ***
***                                                                ***
***     perl configdata.pm --dump                                  ***
***                                                                ***
**********************************************************************
: levitte@lapdog:~/gitwrk/openssl.net/official/_build
: ; rm -rf tmp; mkdir tmp; for s in 1 5; do ../master/util/process_docs.pl --destdir=tmp --type=man --section=$s; done
tmp/man1/asn1parse.1
tmp/man1/openssl-asn1parse.1 -> tmp/man1/asn1parse.1
tmp/man1/CA.pl.1
tmp/man1/ca.1
tmp/man1/openssl-ca.1 -> tmp/man1/ca.1
tmp/man1/ciphers.1
tmp/man1/openssl-ciphers.1 -> tmp/man1/ciphers.1
tmp/man1/cms.1
tmp/man1/openssl-cms.1 -> tmp/man1/cms.1
tmp/man1/crl.1
tmp/man1/openssl-crl.1 -> tmp/man1/crl.1
tmp/man1/crl2pkcs7.1
tmp/man1/openssl-crl2pkcs7.1 -> tmp/man1/crl2pkcs7.1
tmp/man1/dgst.1
tmp/man1/openssl-dgst.1 -> tmp/man1/dgst.1
tmp/man1/dhparam.1
tmp/man1/openssl-dhparam.1 -> tmp/man1/dhparam.1
tmp/man1/dsa.1
tmp/man1/openssl-dsa.1 -> tmp/man1/dsa.1
tmp/man1/dsaparam.1
tmp/man1/openssl-dsaparam.1 -> tmp/man1/dsaparam.1
tmp/man1/ec.1
tmp/man1/openssl-ec.1 -> tmp/man1/ec.1
tmp/man1/ecparam.1
tmp/man1/openssl-ecparam.1 -> tmp/man1/ecparam.1
tmp/man1/enc.1
tmp/man1/openssl-enc.1 -> tmp/man1/enc.1
tmp/man1/engine.1
tmp/man1/openssl-engine.1 -> tmp/man1/engine.1
tmp/man1/errstr.1
tmp/man1/openssl-errstr.1 -> tmp/man1/errstr.1
tmp/man1/gendsa.1
tmp/man1/openssl-gendsa.1 -> tmp/man1/gendsa.1
tmp/man1/genpkey.1
tmp/man1/openssl-genpkey.1 -> tmp/man1/genpkey.1
tmp/man1/genrsa.1
tmp/man1/openssl-genrsa.1 -> tmp/man1/genrsa.1
tmp/man1/list.1
tmp/man1/openssl-list.1 -> tmp/man1/list.1
tmp/man1/nseq.1
tmp/man1/openssl-nseq.1 -> tmp/man1/nseq.1
tmp/man1/ocsp.1
tmp/man1/openssl-ocsp.1 -> tmp/man1/ocsp.1
tmp/man1/openssl.1
tmp/man1/passwd.1
tmp/man1/openssl-passwd.1 -> tmp/man1/passwd.1
tmp/man1/pkcs12.1
tmp/man1/openssl-pkcs12.1 -> tmp/man1/pkcs12.1
tmp/man1/pkcs7.1
tmp/man1/openssl-pkcs7.1 -> tmp/man1/pkcs7.1
tmp/man1/pkcs8.1
tmp/man1/openssl-pkcs8.1 -> tmp/man1/pkcs8.1
tmp/man1/pkey.1
tmp/man1/openssl-pkey.1 -> tmp/man1/pkey.1
tmp/man1/pkeyparam.1
tmp/man1/openssl-pkeyparam.1 -> tmp/man1/pkeyparam.1
tmp/man1/pkeyutl.1
tmp/man1/openssl-pkeyutl.1 -> tmp/man1/pkeyutl.1
tmp/man1/prime.1
tmp/man1/openssl-prime.1 -> tmp/man1/prime.1
tmp/man1/rand.1
tmp/man1/openssl-rand.1 -> tmp/man1/rand.1
tmp/man1/rehash.1
tmp/man1/openssl-c_rehash.1 -> tmp/man1/rehash.1
tmp/man1/openssl-rehash.1 -> tmp/man1/rehash.1
tmp/man1/c_rehash.1 -> tmp/man1/rehash.1
tmp/man1/req.1
tmp/man1/openssl-req.1 -> tmp/man1/req.1
tmp/man1/rsa.1
tmp/man1/openssl-rsa.1 -> tmp/man1/rsa.1
tmp/man1/rsautl.1
tmp/man1/openssl-rsautl.1 -> tmp/man1/rsautl.1
tmp/man1/s_client.1
tmp/man1/openssl-s_client.1 -> tmp/man1/s_client.1
tmp/man1/s_server.1
tmp/man1/openssl-s_server.1 -> tmp/man1/s_server.1
tmp/man1/s_time.1
tmp/man1/openssl-s_time.1 -> tmp/man1/s_time.1
tmp/man1/sess_id.1
tmp/man1/openssl-sess_id.1 -> tmp/man1/sess_id.1
tmp/man1/smime.1
tmp/man1/openssl-smime.1 -> tmp/man1/smime.1
tmp/man1/speed.1
tmp/man1/openssl-speed.1 -> tmp/man1/speed.1
tmp/man1/spkac.1
tmp/man1/openssl-spkac.1 -> tmp/man1/spkac.1
tmp/man1/srp.1
tmp/man1/openssl-srp.1 -> tmp/man1/srp.1
tmp/man1/storeutl.1
tmp/man1/openssl-storeutl.1 -> tmp/man1/storeutl.1
tmp/man1/ts.1
tmp/man1/openssl-ts.1 -> tmp/man1/ts.1
tmp/man1/tsget.1
tmp/man1/openssl-tsget.1 -> tmp/man1/tsget.1
tmp/man1/verify.1
tmp/man1/openssl-verify.1 -> tmp/man1/verify.1
tmp/man1/version.1
tmp/man1/openssl-version.1 -> tmp/man1/version.1
tmp/man1/x509.1
tmp/man1/openssl-x509.1 -> tmp/man1/x509.1
tmp/man5/config.5
tmp/man5/x509v3_config.5
: levitte@lapdog:~/gitwrk/openssl.net/official/_build
: ; for x in tmp/man*/*.[15]; do LC_ALL=en_US.UTF-8 MANROFFSEQ='' MANWIDTH=80 man --warnings -E UTF-8 -l -Tutf8 -Z $x 2>&1 >/dev/null | sed -e "s|^<standard input>|$x|"; done
tmp/man1/ciphers.1:230: warning [p 2, 2.2i]: can't break line
tmp/man1/dsa.1:212: warning [p 1, 10.0i, div `an-div', 0.0i]: can't break line
tmp/man1/gendsa.1:173: warning [p 1, 4.5i, div `an-div', 0.0i]: can't break line
tmp/man1/genrsa.1:180: warning [p 1, 4.7i, div `an-div', 0.0i]: can't break line
tmp/man1/openssl-ciphers.1:230: warning [p 2, 2.2i]: can't break line
tmp/man1/openssl-dsa.1:212: warning [p 1, 10.0i, div `an-div', 0.0i]: can't break line
tmp/man1/openssl-gendsa.1:173: warning [p 1, 4.5i, div `an-div', 0.0i]: can't break line
tmp/man1/openssl-genrsa.1:180: warning [p 1, 4.7i, div `an-div', 0.0i]: can't break line
tmp/man1/openssl-rsa.1:213: warning [p 1, 9.7i, div `an-div', 0.0i]: can't break line
tmp/man1/rsa.1:213: warning [p 1, 9.7i, div `an-div', 0.0i]: can't break line
: ; ls tmp/man*
tmp/man1:
asn1parse.1          openssl-ca.1         openssl-pkey.1       pkcs7.1
ca.1                 openssl-ciphers.1    openssl-pkeyparam.1  pkcs8.1
CA.pl.1              openssl-cms.1        openssl-pkeyutl.1    pkey.1
ciphers.1            openssl-c_rehash.1   openssl-prime.1      pkeyparam.1
cms.1                openssl-crl.1        openssl-rand.1       pkeyutl.1
c_rehash.1           openssl-crl2pkcs7.1  openssl-rehash.1     prime.1
crl.1                openssl-dgst.1       openssl-req.1        rand.1
crl2pkcs7.1          openssl-dhparam.1    openssl-rsa.1        rehash.1
dgst.1               openssl-dsa.1        openssl-rsautl.1     req.1
dhparam.1            openssl-dsaparam.1   openssl-s_client.1   rsa.1
dsa.1                openssl-ec.1         openssl-sess_id.1    rsautl.1
dsaparam.1           openssl-ecparam.1    openssl-smime.1      s_client.1
ec.1                 openssl-enc.1        openssl-speed.1      sess_id.1
ecparam.1            openssl-engine.1     openssl-spkac.1      smime.1
enc.1                openssl-errstr.1     openssl-srp.1        speed.1
engine.1             openssl-gendsa.1     openssl-s_server.1   spkac.1
errstr.1             openssl-genpkey.1    openssl-s_time.1     srp.1
gendsa.1             openssl-genrsa.1     openssl-storeutl.1   s_server.1
genpkey.1            openssl-list.1       openssl-ts.1         s_time.1
genrsa.1             openssl-nseq.1       openssl-tsget.1      storeutl.1
list.1               openssl-ocsp.1       openssl-verify.1     ts.1
nseq.1               openssl-passwd.1     openssl-version.1    tsget.1
ocsp.1               openssl-pkcs12.1     openssl-x509.1       verify.1
openssl.1            openssl-pkcs7.1      passwd.1             version.1
openssl-asn1parse.1  openssl-pkcs8.1      pkcs12.1             x509.1

tmp/man5:
config.5  x509v3_config.5
```

**Comment by richsalz** [comment]
I found errors, you didn't.  I removed the man5/config changes to address your concerns.  Any recommendation for next step?

File: doc/man1/ts.pod [hunk 6]
```diff
@@ -557,10 +557,12 @@ OID section of the config file):
...
-without any other key usage extensions. You can add the
-'extendedKeyUsage = critical,timeStamping' line to the user certificate section
-of the config file to generate a proper certificate. See L<req(1)>,
-L<ca(1)>, L<x509(1)> for instructions. The examples
+without any other key usage extensions. You can add this line to the
+user certificate section of the config file to generate a proper certificate;
+
+   extendedKeyUsage = critical,timeStamping
+
+See L<req(1)>, L<ca(1)>, L<x509(1)> for instructions. The examples
```
Comment (levitte) [inline doc/man1/ts.pod]: The last L should be preceeded with an "and"

**Comment by richsalz** [comment]
@levitte discovered that I was running man on the pod pages, not the generated man pages.  Sorry to waste everyone's time, especially his.  Sheesh.

File: doc/man5/x509v3_config.pod (hunk 5 above)
Comment (kaduk) [reply doc/man5/x509v3_config.pod]: > Having multiple email addresses in the SAN is confusing

Why is having multiple email addresses in the SAN confusing?

**Pull request closed without merging**